runtime/doc/usr_05.txt

```diff
@@ -250,121 +250,115 @@ The ":map" command (with no arguments) lists your current mappings.  At
+
+There are mainly two kinds of plugin: global and filetype-specific.
+
+Global plugins are sourced on startup and usually add functionality that will
+be available regardless of the opened files.  For example, the |matchparen|
+plugin can be used in any kind of file whatsoever to highlight matching
+parentheses on cursor movement.
+					                *standard-plugin*
 When you start Vim, it will automatically load a number of global plugins.
 You don't have to do anything for this.  They add functionality that most
 people will want to use, but which was implemented as a Vim script instead of
 being compiled into Vim.  You can find them listed in the help index
 |standard-plugin-list|.  Also see |load-plugins|.
 
-							*add-global-plugin*
-You can add a global plugin to add functionality that will always be present
-when you use Vim.  There are only two steps for adding a global plugin:
-1. Get a copy of the plugin.
-2. Drop it in the right directory.
+Global plugins reside under the `plugin/` |vimfiles| directory.
+
+                                                        *ftplugins*
+Filetype plugins ("ftplugins") are only used for a specific type of file, and
+usually change the buffer-local settings.  For example, the standard vim C
+ftplugin sets 'formatoptions' to break comment lines but not other lines, and
+to insert the comment leader when hitting <CR> or using |o|.  Filetype plugins
+are usually distributed alongside syntax files, and indent and filetype
+detection scripts to provide support for aditional programming or markup
+languages in Vim.
+
+Filetype plugins reside under the `ftplugin/` |vimfiles| directory.
+
+The Vim distribution comes with a set of plugins for different filetypes that
+you can start using with this command: >
+
+	:filetype plugin on
+
+See |vimrc-filetype|.
 
+GETTING PLUGINS                                         *plugins-getting*
 
-GETTING A GLOBAL PLUGIN
+There are many places to get plugins from:
 
-Where can you find plugins?
 - Some come with Vim.  You can find them in the directory $VIMRUNTIME/macros
   and its sub-directories.
-- Download from the net.  There is a large collection on http://www.vim.org.
+- In the net:
+  - A nicely curated list of plugins is available at http://vimawesome.com/.
+  - There is a large collection of plugins on http://www.vim.org/scripts.
+  - Github is very popular for plugin creators. See
+    https://github.com/search?q=language%3AVimL&type=Repositories
 - They are sometimes posted in a Vim |maillist|.
-- You could write one yourself, see |write-plugin|.
+- You could write one yourself ;) See |write-plugin|.
+
+INSTALLING PLUGINS                   *add-global-plugin*  *add-filetype-plugin*
 
 Some plugins come as a vimball archive, see |vimball|.
 Some plugins can be updated automatically, see |getscript|.
 
+For all other cases, you should be able to follow the guidelines below (also,
+it's very likely that the plugin creators has given instructions on how to
```

Comment:
Should read:

it's very likely that the plugins creators *have* given instructions on how...